### PR TITLE
adding documentation for the yandex_transport component

### PR DESCRIPTION
**Pull request by rishatik92**
https://github.com/home-assistant/home-assistant/pull/26252#discussion_r321543743

**Description:**
Documentation for Yandex Transport component:
https://github.com/home-assistant/home-assistant/pull/26252#discussion_r321543743 

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/yandex_transport.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Yandex transport"
+description: "Instructions on how to setup Yandex transport with Home Assistant."
+logo: yandex.png
+ha_category:
+  - Transport
+ha_release: 0.99
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/yandex_transport.markdown]: Please remove the `redirect_from`

File: source/_components/yandex_transport.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Configuration
+
+To activate Yandex Transport, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: yandex_tranport
+    stop_id: YOUR_STOP_ID
+    routes: #optionaly
```
Comment (klaasnicolaas) [inline source/_components/yandex_transport.markdown]: In the first example only the required items

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/yandex_transport.markdown (hunk 1 above)
Comment (rishatik92) [reply source/_components/yandex_transport.markdown]: you mean to remove "redirect_from:
 \- /components/sensor.yandex_tranport/" , with _- /components/sensor.yandex_tranport/_?

Comment (klaasnicolaas) [reply source/_components/yandex_transport.markdown]: yes both lines

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

File: source/_components/yandex_transport.markdown [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+---
+
+The `yandex_tranport` sensor platform uses [Yandex Maps](https://maps.yandex.ru/) it will give you the time until the next departure time from a bus/tramway/etc stop.
+
+The [Yandex Maps](https://maps.yandex.ru/) website can help to determine the id of your bus stop. You can select bus stop by clicking on the map, and look to the url:
+
+https://yandex.ru/maps/213/moscow/?ll=37.722565%2C55.806662&masstransit%5BstopId%5D=stop__9642962&mode=masstransit&z=16.52
+where stop id is: **9642962**
+
+if you want to look only specific routes, you can add them in routes section.
```
Comment (MartinHjelmare) [inline source/_components/yandex_transport.markdown]: If you want to track only specific routes, you can add them in the routes section.

Comment (rishatik92) [reply source/_components/yandex_transport.markdown]: fixed

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @rishatik92! 👍

**Pull request merged**